content/guides/kubernetes/observability-k8s-monitoring-checklist.md

```diff
@@ -0,0 +1,451 @@
+---
+title: "Kubernetes Monitoring Checklist"
+linkTitle: "Kubernetes Monitoring Checklist"
+description: "A list of components that platform operators should monitor"
+parent: "Platform Observability"
+weight: 1600
+keywords:
+- Kubernetes
+---
+
+To run a Kubernetes platform effectively, cluster administrators need visibility
+into the behavior of the system. Furthermore, platform administrator need to be
+alerted when any of the critical platform components are unavailable or behaving
+erratically.
+
+This guide provides a list of components that platform operators should monitor.
+More importantly, it lists important conditions that operators should use to
+generate alerts. The guide does not prescribe a specific monitoring or alerting
+tool. Instead, it focuses on _what_ to monitor.
+
+## How to Use This Guide
+
+The sections below outline _conditions_ that platform operators must monitor when
+operating Kubernetes. For each condition, the guide provides the following:
+
+- _Threshold:_ The magnitude that must be exceeded to generate an alert.
+- _Severity:_ The severity of the condition, given the threshold is met.
+- _Metrics:_ The available metrics to monitor.
+- _Notes:_ Additional information applicable to the condition.
+
+If the condition is true and above the given threshold, the monitoring system
+should generate an alert with the given severity.
+
+It is important to keep in mind that thresholds and the severity of alerts will
+vary for each environment. Platform operators can use this guide as a starting
+point for their monitoring implementation.
+
+## etcd
+
+### Member Down
+
+- _Threshold:_ 3 minutes
+- _Severity:_ Critical
+- _Notes:_ A single member failure does not have a direct impact on the
+  Kubernetes cluster. However, it increases the risk of experiencing etcd quorum
+  loss if additional members fail.  
+
+### Majority of Members Down
```

Comment:
Just being nitpicky and I think it's fine as is, but might be worth considering making these H4 headers instead of H3, since the current styling makes it a bit hard to differentiate between H3 and H2. Up to you though!

Reply:
If that's the case, then I'm a big proponent of fixing the styling, not changing which headers we use because we like the look of one better than the other. Getting the headers and semantic structure of the document right is important for accessibility.

Reply:
Opened an issue for this